web/README.md

```diff
@@ -1,10 +1,28 @@
-# VAST Docs
+# VAST Website
 
-This directory the [official VAST documentation][site], along with the
-[Docusaurus](https://docusaurus.io/) scaffold for local building, viewing, and
-deploying.
+This directory contains code and content of the official VAST website at
+https://vast.io. We use [Docusaurus](https://docusaurus.io/) as driving
```

Comment:
Should we generally write links as `[vast.io](https://vast.io)` instead as that reads a bit nicer?

Reply:
I think it depends on the context. Here, the URI scheme helps understand where you land.